docs/triage.md

```diff
@@ -0,0 +1,90 @@
+# Issue/Pull Request Management
+
+## Overview
+The Project Reunion team manages issues and PRs in the repo using a process we call
+"triage". It helps us keep issues organized and focuses the attention of the different
+groups that work in our project.
+
+Our objective is to make sure all issues have one or more owners, an issue type label,
+and one or more area labels. New issues are automatically labeled needs-triage by the msftbot,
+so the triage team only needs to review any issues that have been labeled as `needs-triage`.
+
+## Issue Types
+Each issue should only have one type.
```

Comment:
Is it possible for GitHub to enforce this? Is there a way to easily query "show all issues with 2+ types"?

Reply:
I cannot find a query to find issues with 2+ types🤔. I don't believe this would become to big a problem. The bot could help here if needed.